optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/employee/Employee.java

```diff
@@ -44,7 +47,17 @@
     @Size(min = 1, max = 120)
     @Pattern(regexp = "^(?!\\s).*(?<!\\s)$", message = "Name should not contain any leading or trailing whitespaces")
     private String name;
-
+    
+    @NotNull
+    @Size(min = 1, max = 3)
+    @Pattern(regexp = "^(?!\\s).*(?<!\\s)$", message = "Name should not contain any leading or trailing whitespaces")
+    private String shortId;
+    
+    @NotNull
+    @Size(min = 7, max = 7)
+    @Pattern(regexp = "^#[0-9A-F][0-9A-F][0-9A-F][0-9A-F][0-9A-F][0-9A-F]$")
+    private String color;
```

Comment:
I don't think this belongs to the domain model. Unless I'm missing something, the color and shortId are only needed for the rotation editor, which is strictly a UI feature. Color and shortID should go to frontend to keep the code cohesion high.

Reply:
@ge0ffrey advocated for it so that employee color/short id could also be used in Shift screens